robot/lib/CMakeLists.txt

```diff
@@ -9,8 +9,9 @@ add_library(firm-lib
   Src/drivers/ISM43340.cpp
   Src/drivers/KickerBoard.cpp
   Src/drivers/MCP23017.cpp
-  Src/drivers/MPU6050.cpp)
-
+  Src/drivers/MPU6050.cpp
```

Comment:
Can we remove this?

Reply:
I'd rather not remove drivers for devices we no longer use. They can still be useful in cases.